### DISCUSSION TITLE
写了一篇nexttrace深度剖析以及用法总结，涵盖了各个参数

### DISCUSSION THREAD

**Discussion by RokasUrbelis** [nxtrace/NTrace-core#170] [General]
同时也写了一篇nexttrace深度剖析以及用法总结，涵盖了各个参数，希望可以帮助到大家，如果有帮助希望可以推广推广帮助到更多人

文章链接：https://cloud.tencent.com/developer/article/old/2264465

**Comment by xqzr** [comment]
https://github.com/nxtrace/Ntrace-V1/discussions
